lib/modules/showImages.js

```diff
@@ -1400,13 +1400,16 @@ function setMediaMaxSize(media) {
 }
 
 function addDragListener({ media, atShiftKey, onStart, onMove }) {
-	let isActive, hasMoved, lastX, lastY;
+	let isActive, isReleased, hasMoved, lastX, lastY;
 
 	const handleMove = frameDebounce(e => {
-		if (!e.movementX && !e.movementY) {
+		if (
+			process.env.BUILD_TARGET !== 'safari' && // Safari lacks e.movement(X|Y)
```

Comment:
This may sometimes cause an issue where clicking on the image does not open it, but I don't know if this condition (!e.movementX && !e.movementY) is ever met in Safari.